docs/introduction/models.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "3D Models"
+type: introduction
+layout: docs
+parent_section: introduction
+order: 8.75
+---
+
+*work in progress*
+
+## Model File Formats
+
+*work in progress*
+
+## Animating Models
+
+*work in progress*
+
+## Where to Find Models
+
+*work in progress*
+
+## How to Create Models
+
+*work in progress*
+
+## Troubleshooting
+
+### I Don't See My Model
+
+Check your console for errors. If there are no errors, try scaling your model
+down. Often times there's a mismatch in the scale when you export, and this
+will cause the camera to be inside the model, which means you won't be able to
+see it.
+
+```html
+<a-entity gltf-model="#tree" scale="0.01 0.01 0.01"></a-entity>
+```
+
+If this doesn't work, open the Inspector by pressing `ctrl + alt + i` and zoom
+out to verify the model is actually there.
+
+### I Don't See Textures
+
+Sometimes textures just won't work right off the bat. This is usually because
+exporters use absolute paths for textures, which won't work on the Web:
+
+1. Open the model (or .mtl if you're doing OBJ) in a plain text editor
+2. Search for the name of your texture (e.g., `texture.jpg`)
+3. Fix the path to the texture by making it relative instead of absolute
+
+### My Model Isn't Animating
+
+Don McCurdy has written the animation-mixer component in his aframe-extras
+repository to provide an API for running model animations.
```

Comment:
Proposed:

> The [`animation-mixer` component](https://github.com/donmccurdy/aframe-extras/tree/master/src/loaders#animation), part of [aframe-extras](https://github.com/donmccurdy/aframe-extras) by Don McCurdy, provides controls for playing animations in three.js (`.json`) and glTF (`.gltf`) models.

I'd also be amenable to merging that component into A-Frame core, if we're interested. It's technically compatible with [eight three.js loaders](https://threejs.org/docs/#manual/introduction/Animation-system).